doc/changes/latest.inc

```diff
@@ -48,6 +46,7 @@ Bug
 
 - Fix bug with :func:`mne.io.read_raw_gdf` where birthdays were not parsed properly, leading to an error by `Svea Marie Meyer`_
 
+- Fix bug with :func:`mne.io.read_raw_edf` where recording ID  was not read properly for non-ASCII characters by `Lx37`_
```

Comment:
Still 2 spaces here: `ID  was`

Reply:
Fixed in 138fa5d

@cbrnr Feel free to use the `suggestion` feature for such reviews, speeds up the process :)

Point 6 here
https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/commenting-on-a-pull-request#adding-line-comments-to-a-pull-request

Reply:
OK, will try that next time (I've never noticed it)!